lib/db.js

```diff
@@ -29,7 +29,7 @@ var async = require('async'),
     logger = require('logops'),
     database,
     orionDb,
-    delay = 2000, reportInterval = 15e3, lastErrorTrace = 0;
+    delay = config.checkDB.delay, reportInterval = config.checkDB.reportInterval, lastErrorTrace = 0;
```

Comment:
I don't know where our guide is, I usually trust in "reformat code" and linters run by grunt . This passed ... In any case, fixed 1f0b55b